src/main/java/org/folio/rest/validator/ValidatorUtil.java

```diff
@@ -134,6 +141,17 @@ public static boolean isUrlValid(String url) {
     }
   }
 
+  public static boolean isUUIDValid(String uuid) {
```

Comment:
I don't believe there is no any standard UUID validation. Why do we need to invent another one?

Reply:
standard UUID.fromString(id) does not correctly validates uuid(s). For instance the invalid value '99999999-9999-2-9999-999999999999' passes UUID.fromString(id) validation but it is not correct.